src/__tests__/router.test.ts

```diff
@@ -0,0 +1,49 @@
+import {server} from '../server'
+const apiJson = require('../../docs/api.json')
+
+type httpMethod = 
+  "GET"
+  | "POST"
+  | "PUT"
+  | "DELETE"
+
+type routeTestFixture = {
```

Comment:
From what I've seen, convention is that all non-primitive types are capitalised; `RouteTestFixture` and `HttpMethod`.